platform/src/main/java/org/stellar/anchor/platform/config/PropertyClientsConfig.java

```diff
@@ -1,34 +1,30 @@
 package org.stellar.anchor.platform.config;
 
-import static java.util.Collections.emptyList;
-import static java.util.Collections.emptyMap;
 import static org.apache.commons.lang3.ObjectUtils.isEmpty;
 import static org.stellar.anchor.client.DefaultClientService.*;
 import static org.stellar.anchor.util.Log.debugF;
 import static org.stellar.anchor.util.Log.error;
 
 import com.google.common.collect.ImmutableMap;
+import com.google.gson.Gson;
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
 import lombok.Data;
 import org.jetbrains.annotations.NotNull;
 import org.springframework.validation.Errors;
 import org.springframework.validation.Validator;
 import org.stellar.anchor.api.exception.InvalidConfigException;
-import org.stellar.anchor.client.ClientConfig;
-import org.stellar.anchor.client.CustodialClientConfig;
-import org.stellar.anchor.client.NonCustodialClientConfig;
+import org.stellar.anchor.client.*;
 import org.stellar.anchor.config.ClientsConfig;
+import org.stellar.anchor.util.GsonUtils;
 
 @Data
 public class PropertyClientsConfig implements ClientsConfig, Validator {
-  String type;
+  ClientsConfig.ClientsConfigType type;
```

Comment:
nit: Add on-demand static imports
```suggestion
  ClientsConfigType type;
```